web/pdf_page_view.js

```diff
@@ -535,25 +535,28 @@ var PDFPageView = (function PDFPageViewClosure() {
       var CustomStyle = pdfjsLib.CustomStyle;
       var pdfPage = this.pdfPage;
 
-      var viewport = pdfPage.getViewport(1);
+      // increase to improve quality
+      var viewport = pdfPage.getViewport(4);
```

Comment:
Unless I'm mistaken, isn't this making things ever worse from a memory perspective?
What governs the memory usage for a canvas is (basically) its size. With the latest changes it seems that you're increasing the size, and thus memory consumption, by a factor of `4`!